Server/Plugins/APIDump/main_APIDump.lua

```diff
@@ -1696,7 +1696,53 @@ globals =
 		end
 	end
 
-	file:write("}\n")
+	file:write("}\n\n")
+
+	-- Add merge code
+	file:write([[
+-- ## Main ##
+
+function WriteTable(a_File, a_TableName)
+	a_File:write(a_TableName, " = \n{\n")
+	for _, Entry in ipairs(_G[a_TableName]) do
+		a_File:write("\t\"", Entry, "\",\n")
+	end
+	a_File:write("}\n\n")
+end
+
+-- Load plugins's luacheck
+local FilePluginLuacheck = assert(loadfile(".plugin_luacheck"))
```

Comment:
I load a plugin specific one, the merge code is for merging both options / tables into one file.

Not sure what you really mean.